File: src/Backend/src/main/java/de/crystalcase/envcalc/entities/Exchange.java
package de.crystalcase.envcalc.entities;

import lombok.Getter;
import lombok.Setter;
import org.springframework.data.annotation.Id;
import org.springframework.data.elasticsearch.annotations.Document;
import org.springframework.data.elasticsearch.annotations.Field;

import java.util.ArrayList;

@Getter
@Setter
@Document(indexName = "probas")
public class Exchange {

    @Id
    private Flow flow;
    private Unit unit;
    @Field(name = "input")
    private Boolean input;
    @Field(name = "amount")
    private Double amount;

}

File: src/Frontend/EnvCalc.Frontend/EnvCalc.Core/EnvCalc.BusinessObjects/Exchange.cs
namespace EnvCalc.BusinessObjects
{
    public class Exchange
    {
        public string Id { get; set; }

        public string Name { get; set; }

        public string Unit { get; set; }

        public double Amount { get; set; }

        public bool? Input { get; set; }
    }
}